packages/wonder-blocks-core/components/clickable-behavior.test.js

```diff
@@ -511,6 +533,23 @@ describe("ClickableBehavior", () => {
         expect(onClick).toHaveBeenCalledTimes(expectedNumberTimesCalled);
     });
 
+    it("calls onClick on mouseup when the mouse was dragging", () => {
```

Comment:
I think this test needs a `mousedown` to initiate the dragging.  I'm concerned that the way we're simulating events is actually triggering the drag handler.  It would be good to verify that the drag handler is being called as part of this test.  We may end of having to simulate a `dragStart` even ourselves as part of this and the other test(s) that check the drag behavior.